xml/hwbp_registers.xml

```diff
@@ -493,4 +521,58 @@
             in Table~\ref{tab:action}.
         </field>
     </register>
+
+    <register name="Trigger Extra (RV32)" short="textra32" address="0x7a3">
+        This register is accessible as \Rtdatathree when \Ftype is 2, 3, 4, or
+        5.
+
+        <field name="mvalue" bits="31:26" access="R/W" reset="0">
+            An implementation may tie any number of high bits in this field to
+            0.
+        </field>
+        <field name="mselect" bits="25" access="WARL" reset="0">
+            0: Ignore \Fmvalue.
+
+            1 (optional): This trigger will only match if the low bits of
+            \Rmcontext equal \Fmvalue.
+        </field>
+
+        <field name="0" bits="24:18" access="R" reset="0" />
+        <!-- This is space left open for H mode support. -->
+
+        <field name="svalue" bits="17:2" access="R/W" reset="0">
+            An implementation may tie any number of high bits in this field to
+            0.
+        </field>
+        <field name="sselect" bits="1:0" access="WARL" reset="0">
+            0: Ignore \Fmvalue.
+
+            1 (optional): This trigger will only match if the low bits of
```

Comment:
rather than having all these things "optional" can we just say one "optional" thing at the top for this entire register?

Reply:
I've made that change, but I think the other way is more clear.